lib/samsam.js

```diff
@@ -1,17 +1,19 @@
 "use strict";
 
-var deepEqualCyclic = require("./deep-equal");
 var identical = require("./identical");
 var isArguments = require("./is-arguments");
 var isElement = require("./is-element");
 var isNegZero = require("./is-neg-zero");
 var match = require("./match");
+var deepEqualCyclic = require("./deep-equal").use(match);
+var createMatcher = require("./sinon-match");
 
 module.exports = {
     isArguments: isArguments,
     isElement: isElement,
     isNegZero: isNegZero,
     identical: identical,
     deepEqual: deepEqualCyclic,
-    match: match
+    match: match,
+    createMatcher: createMatcher
```

Comment:
MINOR: Alphabetical order like (most of) the others.

Reply:
Well, it wasn't alphabetically. But I added a commit to order them :)